src/features/game/events/landExpansion/fruitPlanted.ts

```diff
@@ -24,24 +23,39 @@ function getHarvestsLeft() {
 }
 
 function getPlantedAt(
-  fruitName: FruitName,
+  fruitSeedName: FruitSeedName,
   collectibles: Collectibles,
   createdAt: number
 ) {
+  if (!fruitSeedName) return createdAt;
```

Comment:
I know this is just a guard and might never happen but I just returned createdAt here, so IF it happens we return a real value.

Reply:
I see